### ISSUE TITLE
Age bias correction

### ISSUE THREAD

**Issue by anbai106** [miargentieri/proteomic-age-ukb#2]
Hi, 

I wonder, in your NM paper, were all the metrics you reported (MAE and Pearson's r) age-bias-corrected or not? Like the number in Supplementary Fig. S1.

/https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-024-03164-7/MediaObjects/41591_2024_3164_MOESM1_ESM.pdf

Also, is the metric in Supplementary Fig. S1 using the entire >2000 proteins or the feature-selected ~200 proteins? Did you observe overfitting when using the >2000 proteins? 

Age bias correction: https://pmc.ncbi.nlm.nih.gov/articles/PMC7049655/

Thanks

**Comment by miargentieri** [comment]
Thanks for your comment. No, the estimates in Fig. S1 were not age bias corrected. Fig. S1 depicts models using all ~3,000 proteins in the dataset. We only used Boruta to perform additional feature selection in the final selected model (LightGBM). We did not observe overfitting in our LightGBM model, which showed good performance in independent data. But as you will see in Fig. S1 the other model types were possibly much more prone to overfitting.
